python_modules/dagster-graphql/dagster_graphql/schema/roots/mutation.py

```diff
@@ -235,6 +236,19 @@ class Meta:
         name = "TerminateRunFailure"
 
 
+class GrapheneTerminateRunUnauthorizedError(graphene.ObjectType):
```

Comment:
Is it useful to have both this and GrapheneTerminateRunFailure? If the frontend doesn't treat them differently, it seems like you could remove this class and use that instead (and the reason it failed is b/c you're unauthorized)